back-nest/src/sсrapers/scrapers.controller.ts

```diff
@@ -23,17 +24,30 @@ export class ScraperController {
   }
 
   @Get('scrape-rozetka')
-  async scrapeRozetka(): Promise<Product[]> {
+  async scrapeRozetka(): Promise<ResponseDto> {
```

Comment:
Use proper naming for all entities in your project. ResponseDTO doesnt tell me anything about that name. I'd recommend you to name it ScrapeRozetkaResponse